cmd/minishift/cmd/root.go

```diff
@@ -123,6 +126,10 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		if hasEnabledExperimental {
+			fmt.Println("!!! Experimental features are enabled !!!")
```

Comment:
-1 for me. What if I for example set "MINISHIFT_ENABLE_EXPERIMENTAL=on" in my bash profile to always have the options available, but then start a Minishift instance without any of the experimental features. In this case "!!! Experimental features are enabled !!!" sounds confusing, since it sounds like something is getting enabled/applied. Maybe "!!! Experimental features are available !!!", but I find this odd as well. Personally I would not print anything here. Maybe as an actual log statement which gets printed when I enable logging, but not per default.